flocker/acceptance/endtoend/test_dataset.py

```diff
@@ -102,3 +108,72 @@ def not_exists():
             return deleted
         created.addCallback(delete_dataset)
         return created
+
+    @skipIf(True,
+            "Shutting down a node invalidates a public IP, which breaks all "
```

Comment:
:grey_question: point of curiosity, have you gotten this to run/pass? Just with static IPs or something?

Just curious because this should have caught bugs such as not using _underlying_blockdevice_api, so I think it does have some value.